Scene.xctemplate/___FILEBASENAME___Coordinator.swift

```diff
@@ -6,9 +6,17 @@
 //  Copyright (c) ___YEAR___ ___ORGANIZATIONNAME___. All rights reserved.
 //
 
-import Foundation
+import UIKit
 import FuntastyKit
 
+fileprivate struct ___FILEBASENAMEASIDENTIFIER___Storyboard: StoryboardType {
+    static let name: String = "___FILEBASENAMEASIDENTIFIER___"
```

Comment:
I think you don't need to explicitly declare the `String` data type as it can be inferred from the value.